packages/fether-react/src/Send/ScanSignedTx/ScanSignedTx.js

```diff
@@ -76,7 +76,7 @@ class ScanSignedTx extends Component {
               <Card className='-centered'>
                 <Scanner
                   onScan={this.onScanSignedTx}
-                  label='Please show the QR code of the signed transaction on the webcam'
+                  label="Show the signed transaction's QR code"
```

Comment:
suggest change to 'Show Parity Signer signed transaction QR code' so the wording is similar to 'Scan Parity Signer account QR code'. Also consider reducing the font size so it fits on single line